Formula/ghc@8.6.rb

```diff
@@ -110,3 +113,25 @@ def post_install
     system "#{bin}/runghc", testpath/"hello.hs"
```

Comment:
Could you use `shell_output` and `assert_match` here to check that `Hellow Homebrew` is being printed?

Reply:
Sure, should I update `ghc` formula as well?

Reply:
Yes please!